reg.py

```diff
@@ -29,8 +30,21 @@
        optimizer = tf.train.AdamOptimizer()             ===> Gives definite values
 '''
 
-def read_dataset():
+def read_dataset(name=None):
+    """
+    read_dataset reads the data set, interpret, and return feature, label extracted data set.
 
+  Args:
+    Nothing
+
+  Returns:
+    `Nump array`, extracted feature columns and label column.
+
+  Example:
+    >>> read_dataset()
+    ( [[2.3, 2.4, 6.5],[2.3, 5.4,3.3]], [12, 82] )
+
+    """
     D = sio.loadmat('Brain_Integ.mat')  
```

Comment:
I remember we talked about using .csv format. Change the reader to read from .csv. Alternatively you can have read_csv and read_mat functions.